s3_file_field/views.py

```diff
@@ -106,20 +163,24 @@ def upload_finalize(request: Request) -> HttpResponseBase:
     # ):
     #     raise BadSignature()
 
-    _multipart.MultipartManager.from_storage(field.storage).finalize_upload(finalization)
+    try:
+        size = _multipart.MultipartManager.from_storage(field.storage).get_upload_size(object_key)
+    except ValueError:
```

Comment:
I think this should be a custom exception type. I could foresee something in the call chain here natively raising its own `ValueError`, and I wouldn't want to mistakenly catch that.